code/game/objects/items/weapons/neutralizer.dm

```diff
@@ -14,10 +14,11 @@
 	icon_state = "neutralizer[active ? "-a" : ""]"
 
 /obj/item/bluespace_neutralizer/attack_self(mob/user)
```

Comment:
I have similar concerns to the above here

Reply:
Whenever supernatural stuff happens in-round, a common and ICly reasonable response is to talk to science about it, and I think that it would be good if science had some things they could actually do in response to the supernatural. Also, constantly veilwalking vampires can become extremely tiresome to deal with, and having a rare but available countermeasure to that is IMO a good idea.